src/DevTools/MockRelayRenderer.tsx

```diff
@@ -18,6 +18,7 @@ import {
   createMockNetworkLayer,
   createMockNetworkLayer2,
 } from "./createMockNetworkLayer"
+import { __webpack_require__ } from "webpack"
```

Comment:
What does this do?

Reply:
Fix a type error, was undefinied... not sure why that was ok before, but this at least quiets down our linters.